projects/plugins/jetpack/modules/search/instant-search/components/dom-event-handler.jsx

```diff
@@ -183,11 +183,13 @@ export default class DomEventHandler extends Component {
 	};
 
 	scrollOverlayToTop = event => {
-		// NOTE: the propertyName need to be aligned with the animation
-		if ( event?.propertyName !== 'opacity' ) {
+		// NOTE: the propertyName need to be aligned with the animation: instant-search/components/overlay.scss.
+		// We don't scroll the overlay on overlay close.
+		if ( event.propertyName !== 'opacity' || ! this.props.isVisible ) {
 			return;
 		}
-		// @see https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo
+
+		// @see https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo
 		window?.scrollTo( 0, 0 );
```

Comment:
We likely want to scroll the overlay element, not the window. See [here](https://github.com/Automattic/jetpack/blame/e76b6ebd438259b44ee2ead459aa4fb3b4410192/projects/plugins/jetpack/modules/search/instant-search/components/search-app.jsx#L170) for how we used to perform overlay scrolls in the past. We won't need to check for overlay visibility using this approach.

Reply:
I looked into the original issue more in-depth and discovered that scrolling the overlay element won't fix our issue here. 😢

Reply:
Thanks for looking into this @jsnmoon